setup.py

```diff
@@ -62,6 +62,7 @@
     long_description=open('README.rst', encoding='utf-8').read(),
     url=package_info['__url__'],
     license=package_info['__license__'],
+    license_files=["LICENSE"],
```

Comment:
It's interesting [setuptools docs](https://setuptools.pypa.io/en/latest/references/keywords.html) say:
> [...] If neither `license_file` nor `license_files` is specified, this option defaults to `LICEN[CS]E*`, [...]

Could be our license file is not included due to older setuptools used.

Anyway, good to be explicit (as we are in most Ocean packages).

Reply:
To be honest, outside of including a license in a pyproject.toml or manifest.in, including licenses has always been flaky for me, apparently setuptools doesn't like them.

Reply:
We usually include it in `setup.cfg`, like [here](https://github.com/dwavesystems/dwave-greedy/blob/master/setup.cfg). That seems to work ok. And it amounts to the same thing you did in this PR.

Reply:
True, that works well too. 
Let me know if you want me to change it.

Reply:
No, this is fine. It's consistent with this repo setup config. We can move to setup.cfg on a bigger refactor.